std_msgs/Header header

# Chessboard Image <-> Pose pair
geometry_msgs/Pose pose
sensor_msgs/Image image